docs/API.md

```diff
@@ -1282,7 +1396,9 @@ Example response:
 {
     "jsonrpc": "2.0",
     "id": 147559211348450,
-    "result": 8
+    "result": {
+      "id": 8
```

Comment:
It would be clearer with "batch_id" instead of just "id". There are other "id". In method "batch_status" "batch_id" is used for the equivalent thing.

Reply:
Rebased and updated to include this proposal.